### PR TITLE
Project-business-site - Maidelin Rubio

### PR DESCRIPTION
**Pull request by rubmaigut**

### CODE REVIEW

File: img/Project 3.html
Comment (EmOlsz) [inline img/Project 3.html:19]: Let's remember that all files as well as folders should be named with lowercase letters - `project` instead of `Project`. Let's fix that ;)

File: img/Project 3.html
Comment (EmOlsz) [inline img/Project 3.html:21]: You anchor is empty, so there is no way of clicking it, we should add some text inside.
Moreover, `href` is not correct - I think you wanted write `href="index.html"`

File: img/Project 3.html
Comment (EmOlsz) [inline img/Project 3.html:39]: Wrong indentations - it should rather be 

```
<button type="submit">
  <i class="fa-search"></i>
  <img />
</button>
```

File: img/Project 3.html
Comment (EmOlsz) [inline img/Project 3.html:58]: capitalised folder name here as well

File: img/Project 3.html
Comment (EmOlsz) [inline img/Project 3.html:74]: We shouldn't use `<br />` tags, let's delete them

File: img/Project 3.html
Comment (EmOlsz) [inline img/Project 3.html:155]: script file should have extension `.js`, without anything else.
There is also another thing - I've noticed you use spanish words from time to time (in class name, or over here). We should avoid that - english language is like a latin of developers, we communicate between countries thanks to that. This rule should be applied to everything we write :)

File: img/Project 3.html
Comment (EmOlsz) [inline img/Project 3.html:158]: Hmm, I don't think you need this. What was the purpose if this code?

File: img/Project 3_files/mobile-style.css
Comment (EmOlsz) [inline img/Project 3_files/mobile-style.css:142]: Let's remember we should style by class names, not by selector - needs to be fixed

File: index.html
Comment (EmOlsz) [inline index.html:8]: I admire your attention to details, but one CSS file is completely enough, we don't have to defragmentize them :)

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Hey Maidelin!
Thank your for your pull request! First thing I wanted to say is that I'm not able to each your website, link is not working. Could you fix that?
Apart from that, in general code needs some fixes. I wrote you some comments, take a look at them and provide changes. I would also like to see your website, so remember to add link :)
Waiting for your reply!

File: img/Project 3.html
Comment (rubmaigut) [reply img/Project 3.html:19]: Hi Max, I forgot to remove this file because it's no used in the project.

File: index.html
Comment (rubmaigut) [reply index.html:8]: Hi Max, 

In this case, is better for me separated the CSS by feature or functionality so, I can have the code organized and this makes it easier for me to understand what I am doing. 

Thanks for your comments I will have it in mind the next time.

**Comment by rubmaigut** [comment]
> Hey Maidelin!
> Thank your for your pull request! First thing I wanted to say is that I'm not able to each your website, link is not working. Could you fix that?
> Apart from that, in general code needs some fixes. I wrote you some comments, take a look at them and provide changes. I would also like to see your website, so remember to add link :)
> Waiti

**Comment by rubmaigut** [comment]
Hi @EmOlsz

I updated the project (readme/files) because I realized that the file that you were reviewing was the wrong one. I forgot to remove it because this was the file where I started to code but then I changed for the index.html that is in the made route of the folder.

This is the link to the live pages. 

https://woodmood-project.netlify.app

File: index.html
Comment (EmOlsz) [inline index.html:95]: `for` and `id` attributes doesn't match over here, let's fix that

File: index.html
Comment (EmOlsz) [inline index.html:104]: When it comes to radio buttons, there is one thing that we should be super aware of : user should be allowed to select only one radio button at a time. In your case however, I was able to select all of them.
How can we fix that? We need to add `name` attribute to all of them, moreover, this attribute has to have same value for all radio buttons, like this :

```
<input type="radio" name="theme" value="furniture" />
<input type="radio" name="theme" value="floaring" />
<input type="radio" name="theme" value="illustration" />
<input type="radio" name="theme" value="mobile-art" />
```

**Comment by EmOlsz** [comment]
Hey Maidelin!
Thank you for your comments, I understand situation better now :)
I've added to extra comments (present above this message), take a look on them and provide changes.
After that, I'll approve project!

File: index.html
Comment (rubmaigut) [reply index.html:104]: Hey, @EmOlsz I've done the changes but after pushing them, I don't see my changes in the PR. Also, it says that the PR is closed. Maybe is because of that?

Comment (EmOlsz) [reply index.html:104]: Hey, indeed, it's closed, you closed it 3 days ago :D If you will reopen it, then we should see the changes!
Screenshot 2021-02-10 at 11 53 41

Comment (rubmaigut) [reply index.html]: HI @EmOlsz 

I didn't realize that was me who closed the PR. Now it's ready.

**Review by EmOlsz — APPROVED** [review]

**Comment by EmOlsz** [comment]
Thank you, project approved :)

**Pull request closed without merging**
